stable/rabbitmq-ha/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.15
-version: 1.27.2
+version: 1.28.0
```

Comment:
Please rebase

Reply:
When opening a new pull request, this is a part of the message:

> When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged.

Doesn't this mean I should not rebase? Or is this referring just  to commits made for PR updates?

Reply:
It just means your changes. I had a PR that was just merged at version 1.28.0 so just rebase that in.